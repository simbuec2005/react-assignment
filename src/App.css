.app {
    padding: 20px;
}

.app-title {
    margin-bottom: 1rem;
}