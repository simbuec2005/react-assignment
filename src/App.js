import React from 'react';
import CustomerContainer from './containers/CustomerContainer';
import './App.css';

function App() {
  return (
    <div className="app">
      <h1 className="app-title">Customer Information</h1>
      <CustomerContainer />
    </div>
  );
}

export default App;
