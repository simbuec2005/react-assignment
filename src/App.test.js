import React from 'react';
import { create } from 'react-test-renderer';
import { Provider } from 'react-redux'
import { createStore } from 'redux'
import App from './App';

const store = createStore(() => { return {}; });

describe('<App />', () => {
  test('Should match snapshot', () => {
    const component = create(<Provider store={store}>
      <App />
    </Provider>);
    expect(component.toJSON()).toMatchSnapshot();
  });
})
