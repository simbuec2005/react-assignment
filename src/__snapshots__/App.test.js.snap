// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Should match snapshot 1`] = `
<div
  className="app"
>
  <h1
    className="app-title"
  >
    Customer Information
  </h1>
  <div
    className="container"
  >
    <div
      className="customet-list"
    >
      <div
        className="customet-list-header"
      >
        <div
          className="col"
        >
          ID
        </div>
        <div
          className="col"
        >
          NAME
        </div>
        <div
          className="col"
        >
          AGE
        </div>
      </div>
    </div>
  </div>
</div>
`;
